src/GoTrueClient.ts

```diff
@@ -238,11 +238,9 @@ export default class GoTrueClient {
    * @param provider One of the providers supported by GoTrue.
    * @param redirectTo A URL to send the user to after they are confirmed (OAuth logins only).
    * @param scopes A space-separated list of scopes granted to the OAuth application.
-   * @param queryParams An object of query params 
+   * @param queryParams An object of query params
    */
-  async signInWithOAuth(
-    credentials: SignInWithOAuthCredentials,
-  ): Promise<OAuthResposne> {
+  async signInWithOAuth(credentials: SignInWithOAuthCredentials): Promise<OAuthResposne> {
```

Comment:
Should this be  `OAuthResponse` ? Not sure if this is done to deconflict with a keyword
```suggestion
  async signInWithOAuth(credentials: SignInWithOAuthCredentials): Promise<OAuthResponse> {
```

Reply:
ah good catch this is unintended, thanks!